contracts/migrations/3_create_sample_listings.js

```diff
@@ -9,12 +9,19 @@ module.exports = function(deployer, network) {
 }
 
 async function deploy_sample_contracts(network) {
-  console.log("deploying listings")
+  let accounts = await new Promise((resolve, reject) => {
```

Comment:
can also just say `let accounts = await web3.eth.getAccounts()`

Reply:
Just tested, that doesn't work. I think it's because we are using Truffle's web3 here, which is version 0.x.

```
Error encountered, bailing. Network state unknown. Review successful transactions manually.
TypeError: Cannot read property '0' of undefined
    at deploy_sample_contracts (/Users/tyler/origin/origin-js/contracts/migrations/3_create_sample_listings.js:14:35)
    at <anonymous>
Using network 'development'.
```

I'm just going to leave it the way it is so that we can do this release.